ingestion/functions/parsing/colombia/colombia.py

```diff
@@ -0,0 +1,215 @@
+            notes = []
+            case = {
+                "caseReference": {
+                    "sourceId": source_id,
+                    "sourceEntryId": entry["ID de caso"],
+                    "sourceUrl": source_url},
+                "demographics": convert_demographics(
+                    entry["Edad"],
+                    entry["Sexo"])}
+            if entry["Fecha diagnostico"] != '':
+                case["events"] = [
+                    {
+                        "name": "confirmed",
+                        "dateRange":
+                        {
+                            "start": convert_date(entry["Fecha diagnostico"]),
+                            "end": convert_date(entry["Fecha diagnostico"])
+                        }
+                    },
+                ]
+            elif entry["Fecha diagnostico"] == '':
+                case["events"] = [
+                    {
+                        "name": "confirmed",
+                        "dateRange":
+                        {
+                            "start": convert_date(entry["fecha reporte web"]),
+                            "end": convert_date(entry["fecha reporte web"])
+                        }
+                    },
+                ]
+                notes.append(
+                    "No Date of Diagnosis provided, so using Date Reported Online (fecha reporte web) as a proxy. This is normally approx. 1 week later.")
+
+            # If patient was symptomatic, mark date of onsetSymptoms, otherwise
+            # asymptomatic
+            if entry["Estado"] == "Asintomático":
+                case["symptoms"] = {
+                    "status": "Asymptomatic",
+                }
+            elif entry["FIS"]:  # maybe change to elif clause and check if can parse field as date
+                case["symptoms"] = {
+                    "status": "Symptomatic",
+                }
+                case["events"].append({
+                    "name": "onsetSymptoms",
+                    "dateRange": {
+                        "start": convert_date(entry['FIS']),
+                        "end": convert_date(entry['FIS']),
+                    }
+                })
+
+            # Include severity of symptoms
+            if entry["Estado"] in ['Leve', 'Moderado', 'Grave']:
+                case["symptoms"]["values"] = [symptom_map.get(entry['Estado'])]
+
+            # Patient Outcome
+            # If patient died, mark date
+            if entry["Fecha de muerte"] != '':
+                case["events"].append({
+                    "name": "outcome",
+                    "value": "Death",
+                    "dateRange": {
+                        "start": convert_date(entry['Fecha de muerte']),
+                        "end": convert_date(entry['Fecha de muerte']),
+                    }
+                })
+                if entry["Estado"] != "Fallecido":
+                    notes.append("Did not die from Covid-19.")
```

Comment:
that should be an event with name "outcome" and value "Recovered"

Reply:
Actually these are cases who died (have a date of death), but did not die of Covid-19. See thread on this in parsing channel

Reply:
Ahh ok, then please say something like "died but not from Covid-19"